plugins/plugin-gdb/che-plugin-gdb-ide/src/main/java/org/eclipse/che/plugin/gdb/ide/GdbDebugger.java

```diff
@@ -65,27 +60,23 @@ public GdbDebugger(DebuggerServiceClient service,
               localStorageProvider,
               messageBusProvider,
               eventBus,
-              fqnResolverFactory,
               activeFileHandler,
               debuggerManager,
-              fileTypeRegistry,
               breakpointManager,
               ID);
         this.appContext = appContext;
     }
 
     @Override
-    protected List<String> fqnToPath(@NotNull Location location) {
+    protected String fqnToPath(@NotNull Location location) {
         CurrentProject currentProject = appContext.getCurrentProject();
         if (currentProject == null) {
-            return Collections.singletonList(location.getTarget());
+            return location.getTarget();
         }
-
-        String projectPath = currentProject.getProjectConfig().getPath();
-        String path = projectPath + "/" + location.getTarget();
-        return Collections.singletonList(path);
+        return currentProject.getProjectConfig().getPath() + "/" + location.getTarget();
```

Comment:
For realize it I need change BreakpointManager. The target of this pull request is  a bug fix, not rework BreakpointManager...